subsys/bluetooth/controller/ll_sw/openisa/lll/lll.c

```diff
@@ -163,10 +161,14 @@ int lll_init(void)
 	return 0;
 }
 
-u8_t lll_entropy_get(u8_t len, void *rand)
+u8_t lll_entropy_get(void *buf, u8_t len)
 {
-	/* entropy_get_entropy_isr(dev_entropy, rand, len, 0); */
-	return 0;
+	return entropy_get_entropy_isr(dev_entropy, buf, len, 0);
```

Comment:
@George-Stefan That's interesting. 'ull_conn' is calling this function for the zephyr chosen entropy, which appears to be rv32m1-trng.
There is no entropy for the SKD in the encryption procedure then?

Reply:
I thought, @dleach02 was supposed to follow up on LL entropy usage and crypto API in zephyr

Reply:
> I thought, @dleach02 was supposed to follow up on LL entropy usage and crypto API in zephyr

I added him as a reviewer, and I'll ping him offline; the gist of it is that the TRNG of VEGA is pretty slow, using it directly is not advisable. IIRC David's changes would allow something like the /dev/rng functionality inside Zephyr, but I'm not sure what's the status of that.